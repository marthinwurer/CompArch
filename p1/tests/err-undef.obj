0 1 10100
1 1 05000
2 1 11101
3 1 00000
100 1 00ace
101 1 00bad
0
