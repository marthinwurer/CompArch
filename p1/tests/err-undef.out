CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 0
000:  10100  LDA   100  A[00000]  B[00000]  XR[000]
001:  05000  ????       A[00ace]  B[00000]  XR[000]

Machine Halted - undefined opcode

Simulated time 10 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
